### PR TITLE
Card job layout

### PR DESCRIPTION
**Pull request by LazarevSlava**
# Changed

- Created Card component
- Created Buttons component
- Added style modules

## Checklist

- [x] Linter checks
- [x] Code style checks

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,8 +25,8 @@
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-react-refresh": "^0.4.5",
     "prettier": "^3.2.5",
-    "stylelint": "^16.2.1",
     "stylelint-config-standard": "^36.0.0",
+    "stylelint-config-recommended-scss": "^14.0.0",
```
Comment (michey85) [inline package.json]: Again? You removed stylelint we need and added scss config? Why?

File: src/components/CardJob.jsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    postedAt,
+    contract,
+    location,
+    featured,
+    languages,
+    role,
+    level,
+  } = props;
+
+  const logoBase = baseUrl + logo.slice(1);
```
Comment (michey85) [inline src/components/CardJob.jsx:20]: Mate, you added images and data.json twice. Can we have only one each?

File: src/components/CardJob.jsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+  return (
+    <div className="card">
+      <img className="img-logo" src={logoBase} alt="logo" />
+      <div className="first-line">
+        <p className="company">{company}</p>
+        {props.new ? <p className="new">NEW!</p> : ''}
```
Comment (michey85) [inline src/components/CardJob.jsx]: You used destructuring for all other props but not this one. Why?

File: src/components/CardJob.jsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
+import { baseUrl } from '../constants';
+import './CardJob.css';
```
Comment (michey85) [inline src/components/CardJob.jsx]: We agreed to use CSS modules, not just CSS.

File: src/components/CardJob.jsx [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+      <div className="first-line">
+        <p className="company">{company}</p>
+        {props.new ? <p className="new">NEW!</p> : ''}
+        {featured ? <p className="featured">FEATURED</p> : ''}
+      </div>
+      <p className="position">{position}</p>
+      <div className="third-line">
+        <p className="postedAt">
+          {postedAt}
+          <span>&middot;</span>
```
Comment (michey85) [inline src/components/CardJob.jsx]: What is it?)

File: src/components/ButtonsLanguages.jsx [hunk 6]
```diff
@@ -0,0 +1,6 @@
+import './ButtonLanguage.css';
+
+function ButtonLanguage({ item }) {
+  return <button className="btn">{item}</button>;
+}
```
Comment (michey85) [inline src/components/ButtonsLanguages.jsx]: We have standard prop name `children` for all cases like this. Use it please.

And it has to be passed to a component like `<MyBtn>My text</MyBtn>` not as a regular prop.

File: src/components/CardJob.jsx [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+        </p>
+        <p className="location">{location}</p>
+      </div>
+      <hr />
+      <div className="btn-block">
+        {<ButtonLanguage key={role} item={role} />}
+        {<ButtonLanguage key={level} item={level} />}
+        {languages.map((item, index) => (
+          <ButtonLanguage key={index} item={item} />
+        ))}
```
Comment (michey85) [inline src/components/CardJob.jsx]: Use children prop here.

File: src/components/Header.jsx [hunk 8]
```diff
@@ -0,0 +1,6 @@
+import './Header.css';
```
Comment (michey85) [inline src/components/Header.jsx]: CSS modules please

File: src/components/ListCards.jsx [hunk 9]
```diff
@@ -0,0 +1,15 @@
+import { CardJob } from './CardJob';
+import data from '../data.json';
```
Comment (michey85) [inline src/components/ListCards.jsx]: I thought we agreed to get this data using fetch.

**Review by michey85 — CHANGES_REQUESTED** [review]

File: src/components/CardJob.jsx (hunk 5 above)
Comment (LazarevSlava) [reply src/components/CardJob.jsx]: iScreen Shoter - Просмотр - 240326100605

File: src/components/CardJob.jsx (hunk 3 above)
Comment (LazarevSlava) [reply src/components/CardJob.jsx]: Because  world 'new' can not be used :)

Comment (michey85) [reply src/components/CardJob.jsx]: This is funny :)
But you always can rename it.
```
const { new: isNew } = props
```

File: src/components/CardJob.jsx (hunk 5 above)
Comment (michey85) [reply src/components/CardJob.jsx]: Can we do it with CSS?

File: src/components/CardJob.jsx (hunk 3 above)
Comment (LazarevSlava) [reply src/components/CardJob.jsx]: I already did it

File: src/components/CardJob.jsx (hunk 2 above)
Comment (LazarevSlava) [reply src/components/CardJob.jsx:20]: Sorry, I don't understand what you mean

Comment (michey85) [reply src/components/CardJob.jsx:20]: We have `/public/images` and `src/assets/images` folders. We don't need both.
The same for `data.json`.

File: src/components/CardJob.jsx [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+      <div className={style.first_line}>
+        <p className={style.company}>{company}</p>
+        {isNew ? <p className={style.new}>NEW!</p> : ''}
+        {featured ? <p className={style.featured}>FEATURED</p> : ''}
+      </div>
+      <p className={style.position}>{position}</p>
+      <div className={style.third_line}>
+        <p className={style.postedAt}>
+          {postedAt}
+          <span className={style.dot}></span>
```
Comment (michey85) [inline src/components/CardJob.jsx:34]: Not like this obviously. But let's leave it as it is.

**Review by michey85 — APPROVED** [review]

**Pull request merged**